components/supervisor/cmd/terminal.go

```diff
@@ -35,7 +36,17 @@ func dialSupervisor() *grpc.ClientConn {
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
 	url := fmt.Sprintf("localhost:%d", cfg.APIEndpointPort)
-	conn, err := grpc.DialContext(ctx, url, grpc.WithInsecure(), grpc.WithBlock())
+	opts := []grpc.DialOption{
```

Comment:
I am not sure what we are fixing it, it is a cli tool for testing which should no be used anywhere?